.github/workflows/globals.php

```diff
@@ -33,6 +33,7 @@
 
 $wgHooks['MediaWikiServices'][] = 'wfOnMediaWikiServices';
 
+// phpcs:ignore MediaWiki.Commenting.FunctionComment.MissingDocumentationPublic
```

Comment:
Is this not ignored by default? This file shouldn't be checked at all I don't think?

Reply:
To do it, I think we would add something like this,
```
	<file>.</file>
	<arg name="extensions" value="php,php5,inc" />
	<arg name="encoding" value="UTF-8" />
	<exclude-pattern>coverage</exclude-pattern>
	<exclude-pattern>doc/html</exclude-pattern>
	<exclude-pattern>vendor</exclude-pattern>
```
https://gerrit.wikimedia.org/g/mediawiki/services/parsoid/deploy/+/ebfb301d6544ebb9407e11bb6fbb1e0dffaa4bde/node_modules/wikimedia-langconv/.phpcs.xml#6

But I don't think it is a good idea to exclude the whole globals.php.
Which one do you prefer?

Reply:
It's fine out it is.